docs/docsite/rst/user_guide/collections_using.rst

```diff
@@ -42,6 +42,11 @@ Installing a collection from a git repository
 
 .. include:: ../shared_snippets/installing_collections_git_repo.txt
 
+Installing a collection from source files
+-----------------------------------------
+
+.. include:: ../shared_snippets/installing_collections_file.rst
+
```

Comment:
Hey @lilatomic sorry I didn't notice this before. Can you move this new section below the `Install multiple collections with a requirements file` section please? That way we've introduced the requirements file before talking about your new examples.

Reply:
good call, makes sense. I reordered the (applicable) sections in user_guide/collections_using to follow those of galaxy/user_guide, lmk if that's good.